webapp/app/[locale]/get-started/_components/addChainManually.tsx

```diff
@@ -40,8 +40,8 @@ export const AddChainManually = function ({ chain, layer }: Props) {
   return (
     <div className="border-neutral/55 flex flex-col rounded-xl border border-solid p-4 text-sm font-medium">
       <div className="flex flex-row gap-x-1">
-        <ChainLogo chainId={chain.id} />
-        <span className="ml-1 text-neutral-950">{chain.name}</span>
+        <div className="w-7"><ChainLogo chainId={chain.id} /></div>
+        <span className="text-neutral-950">{chain.name}</span>
```

Comment:
Same here as in `webapp/app/[locale]/get-started/_components/addChainAutomatically.tsx`